### PR TITLE
Reconnect ethclient on repeated error

### PR DESCRIPTION
**Pull request by hkalodner**
This patch should hopefully help with issues where ethclient connections get stuck in a broken state

### CODE REVIEW

**Review by PlasmaPower — DISMISSED** [review]
LGTM

File: packages/arb-util/ethutils/client.go [hunk 1]
```diff
@@ -78,12 +78,17 @@ func NewRPCEthClient(url string) (*RPCEthClient, error) {
 func (r *RPCEthClient) reconnect() error {
 	r.Lock()
 	defer r.Unlock()
+	if r.errCount < maxErrCount {
```
Comment (PlasmaPower) [inline packages/arb-util/ethutils/client.go:81]: Hmm.. technically, this (and the set below) need to also be atomic as the error count is atomically written to outside the mutex.

Comment (hkalodner) [reply packages/arb-util/ethutils/client.go]: Ah whoops. Good catch, I hadn't been thinking. Fix pushed

**Review by PlasmaPower — APPROVED** [review]
LGTM

**Pull request merged**